qlib/backtest/position.py

```diff
@@ -199,14 +203,72 @@ class Position(BasePosition):
     }
     """
 
-    def __init__(self, cash=0, position_dict={}):
+    def __init__(self, start_time, freq, cash: float = 0, position_dict: Dict[str, Dict[str, float]] = {}):
```

Comment:
Force uses to input `start_time, freq` will make it still inconvenient.
Will it be more convenient if it automatically updates the price at the first update_daily_end?